Update Marketing Q3 objective and goals (#6513)
PostHog/posthog.com: contents/handbook/small-teams/marketing/objectives.mdx

* **Objective**:
    * Become an authority and vital resource on building successful products.

* **Goals:**
    * Nail Twitter for content distribution (**Owner:** Andy)
    * Make PostHog for Startups the default choice for technical founders (**Owner:** Joe)

>     * Scale the newsletter to thousands of engaged subscribers (**Owner:** Andy)

Reviewer: ```suggestion
    * Scale the newsletter to thousands of engaged subscribers (**Owners:** Andy & Ian)
```

>     * Create a hub of non-PostHog product engineering content (**Owner:** Andy)

Reviewer: ```suggestion
    * Create a hub of non-PostHog product engineering content (**Owner:** Andy & Ian)
```

Reviewer 2: This one can just be me, if that is good with you

Reviewer: Yup

    * Ship printed PMF guide with a distribution plan (**Owner:** Lottie)

>     * Improve the docs (**Owners:** Lior)

Reviewer: ```suggestion
    * Improve the docs and tutorials (**Owners:** Lior & Ian)
```

Reviewer 2: This one can just be Lior
